lms/djangoapps/teams/startup.py

```diff
@@ -0,0 +1,22 @@
+"""
+Initialize the mako template lookup
+"""
+from django.conf import settings
+
+from edxmako import LOOKUP
```

Comment:
I think the right thing to do here is to change this code: https://github.com/edx/edx-platform/blob/master/common/djangoapps/edxmako/startup.py#L15-L16 so that instead of looking for a fixed set of mako directories defined in settings.py, it uses the normal django template lookup hierarchy: https://docs.djangoproject.com/en/1.4/ref/settings/#template-loaders

An even nicer (but perhaps longer winded) approach would be to use a custom template loader that loaded mako templates: https://docs.djangoproject.com/en/1.4/ref/templates/api/#using-an-alternative-template-language